src/box/sql/trigger.c

```diff
@@ -321,18 +321,13 @@ vdbe_code_drop_trigger(struct Parse *parser, const char *trigger_name,
 void
 sql_drop_trigger(struct Parse *parser)
 {
-	struct drop_entity_def *drop_def = &parser->drop_trigger_def.base;
-	struct alter_entity_def *alter_def = &drop_def->base;
-	assert(alter_def->entity_type == ENTITY_TYPE_TRIGGER);
-	assert(alter_def->alter_action == ALTER_ACTION_DROP);
-	struct SrcList *name = alter_def->entity_name;
-	bool no_err = drop_def->if_exist;
+	bool no_err = parser->drop_object.if_exists;
 
 	struct Vdbe *v = sqlGetVdbe(parser);
 	sqlVdbeCountChanges(v);
 
-	assert(name->nSrc == 1);
-	const char *trigger_name = name->a[0].zName;
+	assert(parser->src_list->nSrc == 1);
+	const char *trigger_name = parser->src_list->a[0].zName;
```

Comment:
Oh no, so `src_list` can be a list of tables or a trigger name? +1 point against merging those request structs and for having one struct per statement with full set of needed members with proper member names.